CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Keycloak Community
+
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services. Keycloak Benchmark is the benchmarking tool supported by the Keycloak Community.
+
+## Building and working with the codebase
+
+Question: Not sure if we should provide more details about this section or have the users pointed to our documentation
+
+## Contributing to keycloak-benchmark
+
+This guide is heavily influenced by the Keycloak's own Contributing guidelines.
+
+Here's a quick checklist for a good PR, more details below:
+
+1. A GitHub Issue with a good description associated with the PR
+2. One feature/change per PR
+3. One commit per PR
+4. PR rebased on main (`git rebase`, not `git pull`)
+5. Good descriptive commit message, with a link to the issue
+6. No changes to code not directly related to your PR
+7. Updates the documentation where necessary
+
+Once you have submitted your PR please monitor it for comments/feedback. We reserve the right to close inactive PRs if
+you do not respond within 4 weeks (bear in mind you can always open a new PR if it is closed due to inactivity).
+
+### Finding something to work on
+
+If you would like to contribute to keycloak-benchmark, but are not sure exactly what to work on, you can find a number of open
+issues that are awaiting contributions in
+[issues](https://github.com/keycloak/keycloak-benchmark/issues).
+
+### Create an issue
+
+Take your time to write a proper issue including a good summary and description.
+
+Remember this may be the first thing a reviewer of your PR will look at to get an idea of what you are proposing
+and it will also be used by the community in the future to find about what new features and enhancements are included in
+new releases.
+
+### Implementing
+
+Do not format or refactor code that is not directly related to your contribution. If you do this it will significantly
+increase our effort in reviewing your PR. If you have a strong need to refactor code then submit a separate PR for the
+refactoring.
+
+### Testing
+
+It would be great to test the changes locally and add a confirmation that it works would be a good show of faith to us. There are also some RightGithub Action based checks for certain components to test the PR.
```

Comment:
There seems to be a typo, I have fixed it in my local commit to remove "Right" from "RightGithub" cc: @ahus1

Reply:
Thanks, updated in latest commit.

Reply:
thanks for fixing it